src/apyfixed.cc

```diff
@@ -995,6 +995,45 @@ APyFixed APyFixed::from_double(
     return result;
 }
 
+APyFixed APyFixed::from_unspecified_double(double value)
```

Comment:
Look at the new `APyFixed.set_from_double()` for properly handling subnormals.

Or better still, let this function only evaluate the necessary width of the resulting APyFixed, and reuse `APyFixed.set_from_double()` for the data casting.

Reply:
Thanks, fixed.
Re-using the function `set_from_double` is certainly good for maintenance, but the function is optimized to generate small word lengths that can accommodate the data while making use of the fact the result is always one or two limbs.